.github/workflows/test.yaml

```diff
@@ -35,19 +35,19 @@ jobs:
         include:
           - sqlite_version: "3440200"
             sqlite_url: "https://www.sqlite.org/2023/sqlite-autoconf-3440200.tar.gz"
-            dart_sdk: 3.4.0
+            dart_sdk: 3.5.0
```

Comment:
Should we update the minimum SDK in the pubspec.yaml as well, or does it still run fine on 3.4.0?

Reply:
Good point, `sqlite3` now requires the newer SDK version and since we depend on the latest `sqlite3` version we should update ours as well.